cognite/client/data_classes/shared.py

```diff
@@ -47,3 +47,94 @@ def __init__(self, count: int = None, value: Union[int, str] = None, **kwargs):
         self.value = value
 
     value = CognitePropertyClassUtil.declare_property("value")
+
+
+class Geometry(dict):
```

Comment:
Shouldn't it be put inside `data_classes/files.py`?

Reply:
Eh, I thought this was cleaner because otherwise the geolocation related classes would have to be defined at the top of `data_classes/files.py` and I thought that made the `data_classes/files.py` file less readable. 

But no, they're technically not shared (yet, at least). No strong opinions on my side so I can move them back if you like.

Reply:
Yeah IMHO it would be better to put it in files.py for as long as they are not being shared anywhere else? :)